README.md

```diff
@@ -58,13 +58,13 @@ sudo dnf install protobuf-compiler
 - A checkout of libbpf
 
 ```shell
-git clone https://github.com/libbpf/libbpf --branch v0.8
+git clone https://github.com/libbpf/libbpf --branch v0.8.0
 ```
 
 ## Building bpfd
 
 ```
-$ cargo xtask build-ebpf --release --libbpf-dir /path/to/libbpf
```

Comment:
@dave-tucker With `--release`, the rest of the commands below need to be updated with `--release` or use `./target/release/...`. Also, see next comment.